tests/Doctrine/MongoDB/Tests/LoggableGridFSTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Doctrine\MongoDB\Tests;
+
+use Doctrine\MongoDB\GridFSFile;
+use Doctrine\MongoDB\LoggableGridFS;
+
+class LoggableGridFSTest extends \PHPUnit_Framework_TestCase
+{
+    const COLLECTION_NAME = 'collectionName';
+    const DATABASE_NAME = 'databaseName';
+
+    public function testLog()
+    {
+        $logResult = false;
+        $loggableGridFS = $this->getLoggableGridFS($this->getLoggerCallable($logResult));
+        $loggableGridFS->log(['test' => 'test']);
+
+        $expected = [
+            'collection' => self::COLLECTION_NAME,
+            'db' => self::DATABASE_NAME,
+            'test' => 'test'
+        ];
+
+        $this->assertEquals($expected, $logResult);
+    }
+
+    public function testStoreFileLog()
+    {
+        $logResult = false;
+
+        $loggableGridFS = $this->getLoggableGridFS($this->getLoggerCallable($logResult));
+
+        $mongoGridFSFile = $this->getMockBuilder('MongoGridFSFile')
```

Comment:
Since we're targeting PHP 5.6+ with this library, you can use `::class` constants instead of specifying class names in strings.